### PR TITLE
PLT-340: test minting policy issue

### PR DESCRIPTION
**Pull request by sjoerdvisscher**
This doesn't actually test that minting policy scripts are executed when calculating fees, but that would basically come down to writing tests for cardano-api, which I don't want to do unless we are sure something is wrong.

Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Formatting, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [ ] Reference the ADR in the PR and reference the PR in the ADR (if revelant)
    - [x] Reviewer requested

### CODE REVIEW

File: plutus-ledger/src/Ledger/Test.hs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE DataKinds           #-}
```
Comment (koslambrou) [inline plutus-ledger/src/Ledger/Test.hs:1]: Why not put this file in the `test` directory?

File: plutus-ledger/src/Ledger/Tx.hs [hunk 2]
```diff
@@ -167,7 +167,21 @@ pattern CardanoApiEmulatorEraTx tx <- (getEmulatorEraTx -> tx) where
 {-# COMPLETE CardanoApiEmulatorEraTx #-}
 
 instance Pretty CardanoTx where
-    pretty = onCardanoTx pretty (pretty . getCardanoApiTxId)
+    pretty tx =
```
Comment (koslambrou) [inline plutus-ledger/src/Ledger/Tx.hs:170]: Why did you need to modify `Pretty` for this PR?

Comment (koslambrou) [reply plutus-ledger/src/Ledger/Tx.hs:170]: Ah, was it to also print the minting policies?

**Review by koslambrou — APPROVED** [review]

File: plutus-ledger/src/Ledger/Test.hs (hunk 1 above)
Comment (sjoerdvisscher) [reply plutus-ledger/src/Ledger/Test.hs:1]: Because it doesn't contain tests, it contains utilities useful for testing. Just like `Plutus.Contract.Test`. The `test` directories we have are not structured in a way that's useable from another package, they are all the same.
